backends/cuda/ceed-cuda.c

```diff
@@ -71,6 +71,8 @@ int CeedCompileCuda(Ceed ceed, const char *source, CUmodule *module,
   char buff[optslen];
   snprintf(buff, optslen,"-arch=compute_%d%d", prop.major, prop.minor);
   opts[numopts + 3] = buff;
+  opts[numopts + 4] = "--maxrregcount=63";
+  opts[numopts + 5] = "--optimization-info=inline";
```

Comment:
This produces confusingly contradictory output
```
remark: default_program:673:29: _Z20log1p_series_shiftedd will not be inlined into CeedKernel_Cuda_ref_ElasFSInitialNH1Diagnostic

remark: default_program:673:29: _Z20log1p_series_shiftedd inlined into CeedKernel_Cuda_ref_ElasFSInitialNH1Diagnostic with cost=-19925 (threshold=1625)
```

Reply:
This was debugging and diagnostics, not a proposal to merge. I just wanted to share what I'd tried. We don't want to dump diagnostics in general, though perhaps when the user requests (say, via an environment variable).